dayrepo/snippets/models.py

```diff
@@ -55,20 +77,193 @@ class Meta(object):
+        verbose_name="走行距離", 
+        blank=False, 
+        null=False
+    )
+
+    def __str__(self):
+
+        return str(self.id)
+
+
+class Snippet(models.Model):
+
+    class Meta(object):
+
+        # 作成されるテーブル名を指定
+        db_table = "snippets"
+        # 管理画面でのモデルの名称
+        verbose_name = "日報"
+
+    # 項目作成
+
+    id = models.AutoField(
+        verbose_name="日報ID",
+        primary_key=True,
+        editable=True,
+        blank=False,
+        null=False,
+    )
+    duties_id = models.ForeignKey(
+        'Duties_trouble', 
+        on_delete=models.DO_NOTHING, 
+        null=False,
+        default=False,
+    )
+    account_id = models.ForeignKey(
+        Account,
+        verbose_name="アカウントID",
+        on_delete=models.DO_NOTHING,
+        null=False,
+        default=False,
+    )
+    start_mileage = models.IntegerField(
+        verbose_name="出発時メーター",
+        blank=False,
+        null=False,
+    )
+    end_mileage = models.IntegerField(
+        verbose_name="到着時メーター",
+        blank=False,
+        null=False,
+    )
+    start_time = models.TimeField(
+        verbose_name="始業時間",
+        blank=False,
+        null=False,
+        max_length=10,
+    )
+    end_time = models.TimeField(
+        verbose_name="終業時間",
+        blank=False,
+        null=False,
+        max_length=10,
+    )
+    start_point = models.CharField(
+        verbose_name="出発地点", 
+        blank=False, 
+        null=False, 
+        max_length=20
+    )
+    end_point = models.CharField(
+        verbose_name="最終到着地点", 
+        blank=False, 
+        null=False, 
+        max_length=20
+    )
+    break_spot = models.CharField(
+        verbose_name="休憩地点", 
+        blank=False, 
+        null=False, 
+        max_length=20
+    )
+    weather = models.CharField(
+        verbose_name="天気",
+        max_length=5,
+        blank=False,
+        null=False,
+    )
+    gasoline_amount = models.FloatField(
+        verbose_name="給油", 
+        max_length=4, 
+        blank=True, 
+        null=True
+    )
+    oil = models.FloatField(
+        verbose_name="オイル", 
+        max_length=4, 
+        blank=True, 
+        null=True
+    )
+    driving_time = models.TimeField(
+        verbose_name="運転時間",
+        blank=False,
+        null=False,
+        max_length=10,
+    )
+    non_driving_time = models.TimeField(
+        verbose_name="運転以外の業務時間",
+        blank=False,
+        null=False,
+        max_length=10,
+    )
+    break_time = models.TimeField(
+        verbose_name="休憩時間",
+        blank=False,
+        null=False,
+        max_length=10,
+    )
+    create_at = models.DateTimeField(
+        verbose_name="作成日時", 
+        auto_now_add=True
+    )
+    update_at = models.DateTimeField(
+        verbose_name="更新日時", 
+        auto_now=True
     )
-    now_mileage = models.IntegerField(verbose_name="走行距離", blank=False, null=False)
 
     def __str__(self):
 
         return str(self.id)
 
+class Duties_trouble(models.Model):
```

Comment:
基本的に、スネークケースとキャメルケースが混ざってしまうのは命名として良くないです。

また、今回はPythonのclass名はアッパーキャメルケースで統一しましょう〜。コーディング規約とかも作った方がいいかもですね。
```suggestion
class DutiesTrouble(models.Model):
```

suggestionを採用する場合、git pull と migrate し直すのをお忘れなく！